### DISCUSSION TITLE
MQTT publish command to switch on/off external LED (GPIO12)

### DISCUSSION THREAD

**Discussion by timostr** [jomjol/AI-on-the-edge-device#2787] [General]
How to make the MQTT publish message to switch on or off the external LED?
What is the "Topic" and how to set the "Payload".

Yes, I have read the documentation before. :-)

**Comment by caco3** [comment]
First, you need to enable the MQTT GPIO subscription for the pin:
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/1783586/5f647cb1-6cb9-423b-96cc-fa595a2a580f)

Then you can send true/false to the topic `<MAINTOPIC>/GPIO/<GPIO NAME>`
